src/dfcx_scrapi/core/agent_assist.py

```diff
@@ -0,0 +1,286 @@
+        creds=None,
+        scope=False,
+        conversation_profile_id: str = None,
+    ):
+        super().__init__(
+            creds_path=creds_path,
+            creds_dict=creds_dict,
+            creds=creds,
+            scope=scope,
+        )
+
+        if conversation_profile_id:
+            self.conversation_profile_id = conversation_profile_id
+
+    def create_conversation_profile(
+        self,
+        display_name,
+        project_id,
+        knowledge_base_id: str = None,
+        suggestion_type: str = "TYPE_UNSPECIFIED",
+        no_small_talk: bool = True,
+        only_end_user: bool = True,
+        max_results: int = 3,
+        language_code: str = "en-US"):
+        """ Create & configure a conversation profile as per the given input.
+
+        Args:
+            display_name (str):
+                Required. Human readable name for this
+                profile. Max length 1024 bytes.
+            project_id (str):
+                Required. The project to create a conversation profile for.
+            knowledge_base_id (str):
+                The knowledge base id of the knowledge base to be configured
+                with the conversation profile.
+            suggestion_type (str):
+                The agent assist feature to be configured with the
+                conversation profile.
+                Allowed values are: ARTICLE_SUGGESTION, FAQ & SMART_REPLY.
+            no_small_talk (bool):
+                Do not trigger if last utterance is small talk.
+            only_end_user (bool):
+                Only trigger suggestion if participant role of last
+                utterance is END_USER.
+            max_results (int):
+                Maximum number of results to return.
+                Currently, if unset, defaults to 10. And the max number is 20.
+            language_code (str):
+                Language code for the conversation profile. If not
+                specified, the language is en-US. Language at
+                ConversationProfile should be set for all non en-us
+                languages. This should be a
+                `BCP-47 <https://www.rfc-editor.org/rfc/bcp/bcp47.txt>`__
+                language tag. Example: "en-US".
+
+        Returns:
+            Conversation profile to be used for incoming
+            Dialogueflow conversations.
+            type: google.cloud.dialogflow_v2beta1.types.ConversationProfile
+        """
+
+        client = services.conversation_profiles.ConversationProfilesClient(
+            credentials = self.creds
+            )
+
+        project_path = client.common_project_path(project_id)
+        logging.info(f"Project Path : {project_path}")
+
+        conversation_profile = types.ConversationProfile()
+        conversation_profile.display_name = display_name
+        conversation_profile.language_code = language_code
+
+        feature_config = types.conversation_profile.HumanAgentAssistantConfig.SuggestionFeatureConfig()
```

Comment:
@hkhaitan1 please run `pylint` on this file as well. There are several lines that exceed the 80-character limit. 

For long lines like this, consider one of the following:
- Update your imports to be more specific
   - instead of all of `types` import the Class you need directly
   - in some situations, this can make your lines shorter in the body of the class, but not always
- Alias long lines
  - Try setting long lines to a reusable alias like `hac = conversation_profile.HumanAgentAssistantconfig`
  - Then you can use `hac` in several places for that method, shortening the lines
- If the above seem infeasible, you can always add `# pylint: disable=C0301` to the end of the line and `pylint` will ignore the `line-too-long` error
  - Try your best to refactor code to fit where possible
  - If not, you can use this 

Consider installing and running an auto code formatter like [black](https://github.com/psf/black) which can be helpful in auto resolving some of these issues.

Reply:
Acknowledged.